### PR TITLE
68 aboutpage quote

### LINKED ISSUE #68: quote block for 'we know what it does...'

### PR DESCRIPTION
**Pull request by RemoLiechti**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #68 

Test URLs:
- Before: https://main--sazerac-wheatleyvodka--hlxsites.hlx.page/about-wheatley
- After: https://68-aboutpageQuote--sazerac-wheatleyvodka--hlxsites.hlx.page/about-wheatley

### CODE REVIEW

**Comment by RemoLiechti** [comment]
review this commit only: https://github.com/hlxsites/sazerac-wheatleyvodka/pull/69/commits/eb760fde13a67beb08cb5d7a8175fb01b83b366d

the others are there due to the branching nature of this adventure

File: blocks/embed/embed.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+main .embed {
```
Comment (RemoLiechti) [inline blocks/embed/embed.css:1]: ignore, changes in base branch 20

File: blocks/embed/embed.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
+/*
```
Comment (RemoLiechti) [inline blocks/embed/embed.js:1]: ignore, changes in base branch 20

File: blocks/faq-replace/faq-replace.css [hunk 3]
```diff
@@ -0,0 +1,78 @@
+.faq-replace {
```
Comment (RemoLiechti) [inline blocks/faq-replace/faq-replace.css:1]: ignore, changes in base branch 20

File: blocks/faq-replace/faq-replace.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
+/**
```
Comment (RemoLiechti) [inline blocks/faq-replace/faq-replace.js:1]: ignore, changes in base branch 20

File: styles/styles.css [hunk 5]
```diff
@@ -26,12 +26,15 @@
   --heading-font-family: 'HCo Ringside Extra Wide', 'HCo Ringside Extra Wide Fallback', sans-serif;
```
Comment (RemoLiechti) [inline styles/styles.css:26]: ignore, changes in base branch 20

File: styles/theme-about.css [hunk 6]
```diff
@@ -0,0 +1,7 @@
+div.background>div>div>div>div:first-child {
```
Comment (RemoLiechti) [inline styles/theme-about.css:1]: ignore, changes in base branch 20

**Review by karlpauls — APPROVED** [review]

**Pull request merged**